com.unity.render-pipelines.universal/Runtime/Passes/DrawObjectsPass.cs

```diff
@@ -71,6 +71,17 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
                 // w is used for knowing whether the object is opaque(1) or alpha blended(0)
                 Vector4 drawObjectPassData = new Vector4(0.0f, 0.0f, 0.0f, (m_IsOpaque) ? 1.0f : 0.0f);
                 cmd.SetGlobalVector(s_DrawObjectPassDataPropID, drawObjectPassData);
+
+                // scaleBias.x = flipSign
+                // scaleBias.y = scale
+                // scaleBias.z = bias
+                // scaleBias.w = unused
+                float flipSign = (renderingData.cameraData.IsCameraProjectionMatrixFlipped()) ? -1.0f : 1.0f;
+                Vector4 scaleBias = (flipSign < 0.0f)
+                    ? new Vector4(flipSign, 1.0f, -1.0f, 1.0f)
+                    : new Vector4(flipSign, 0.0f, 1.0f, 1.0f);
+                cmd.SetGlobalVector(ShaderPropertyId.scaleBiasRt, scaleBias);
```

Comment:
I wonder if someone tries to access these scaleBias variables in passes before drawobjects. Should also set up those at beginning of the frame so any custom pass or pass before main rendering doesn't access garbage/previous frame values?

Can be fixed in follow up PR.